nsis/src/nsis/NsisAnalyzer.java

```diff
@@ -145,4 +148,44 @@ private void resolveStrings(Instruction instr, MemoryBlock stringsBlock)
 		}
 	}
 
+	/**
+	 * Resolve the control flow for the specified instruction
+	 * 
+	 * @param instr        the instruction
+	 * @param entriesBlock the memory block containing the instructions
+	 * @param program
+	 * @throws MemoryAccessException
+	 */
+	private void resolveControlFlow(Instruction instr, MemoryBlock entriesBlock,
+			ReferenceManager referenceManager) throws MemoryAccessException {
+		String mnemonic = instr.getMnemonicString();
+		int instructionNumber;
+		switch (mnemonic) {
+		case "Call":
+			instr.setFlowOverride(FlowOverride.CALL);
+			instructionNumber = instr.getInt(NsisConstants.ARG1_OFFSET);
+			referenceManager.addMemoryReference(instr.getAddress(),
+					entriesBlock.getStart()
+							.add((instructionNumber - 1) * NsisConstants.INSTRUCTION_BYTE_LENGTH),
+					RefType.UNCONDITIONAL_CALL, SourceType.ANALYSIS, NsisConstants.ARG1_INDEX);
+			break;
+
+		case "Jmp":
```

Comment:
I think the Jmp is unconditional and needs to not flow through to the next instruction that follows in the graph view. I recall you need to do that by `instr.setFallThrough(null)`

https://ghidra.re/ghidra_docs/api/ghidra/program/model/listing/Instruction.html#setFallThrough(ghidra.program.model.address.Address)

Reply:
I think that by using `RefType.UNCONDITIONAL_JUMP` in the `addMemoryReference()` function, it already does that. The jump instruction does not continue to the next instruction in the graph view
![image](https://user-images.githubusercontent.com/24403370/89906816-db209600-dbb9-11ea-9eeb-7f2cff27f19b.png)

Do you think calling `instr.setFallThrough(null)` is necessary? The outputs whether it is there or not seem the same in Ghidra

Reply:
I see, donesn't look like it is necessary, good point.

Reply:
I looked back at it, and even though it doesn't make a visible difference in the graph view it makes  a difference when you look at the detailed information of the instruction. It still lists the next address as the fallthrough. So I added `setFallthrough(null)`. Good point